### DISCUSSION TITLE
https://tuxguitar.herac.com.ar/join/

### DISCUSSION THREAD

**Discussion by belfie13** [helge17/tuxguitar#477] [General]
is this the old tuxguitar site? herac i assume is Julian? it won't let me sign up and the sign in page is a 404 error

**Comment by guiv42** [comment]
It's one of the old sites, with tuxguitar.com.ar (which was not maintained)
"herac" is probably linked to [Hernan Andres Casadesus](https://github.com/helge17/tuxguitar/blob/432a6c1ebd60713b5ae92987c26b7daacdc1baec/AUTHORS#L8)

**Reply by belfie13** [reply]
yeah i just tried finding the old developers, no luck, their site is half down, emails don't work any more except for one Nahuel but it's a hotmail account and no response yet. It's going to be a tough one to overcome the stagnant project if they are no longer with us or interested at all.
